### PR TITLE
add(subject): add subject betweenus

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by zainabdnaya — COMMENTED** [review]
i believe  every thing is good

File: subjects/betweenus/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Function
+```go
+func BetweenUs(num, min, max int) bool {
+    // Your code here
+}
+```
+
+
+### Usage
+```go
```
Comment (davidrobert99) [inline subjects/betweenus/README.md:15]: Missing "Here is a possible program to test your function:"

Comment (davidrobert99) [inline subjects/betweenus/README.md:15]: Missing "Here is a possible program to test your function:"

File: subjects/betweenus/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+package main
+
+import "fmt"
+
+func main(){
+    fmt.Println(BetweenUs(1, 2, 3))
+    fmt.Println(BetweenUs(1, 1, 3))
+    fmt.Println(BetweenUs(1, 3, 3))
+    fmt.Println(BetweenUs(1, 1, 1))
+    fmt.Println(BetweenUs(1, 2, 1))
```
Comment (davidrobert99) [inline subjects/betweenus/README.md:25]: Should this be false? Probably it should say that if the min>max the function must return false.

Comment (davidrobert99) [inline subjects/betweenus/README.md:25]: Should this be false? Probably it should say that if the min>max the function must return false.

**Review by davidrobert99 — APPROVED** [review]

**Pull request merged**